pocs/camera/camera.py

```diff
@@ -311,6 +311,42 @@ def set_property(self, prop, val):
         # Forces the command to wait
         self.get_command_result()
 
+    def set_properties(self, prop2index, prop2value):
+        """ Sets a number of properties all at once, by index or value.
+
+        Args:
+            prop2index (dict): A dict with keys corresponding to the property to
+            be set and values corresponding to the index option
+            prop2value (dict): A dict with keys corresponding to the property to
+            be set and values corresponding to the literal value
+        """
+        set_cmd = list()
+        for prop, val in prop2index.items():
+            set_cmd.extend(['--set-config-index', '{}={}'.format(prop, val)])
+        for prop, val in prop2value.items():
+            set_cmd.extend(['--set-config-value', '{}={}'.format(prop, val)])
+
+        self.command(set_cmd)
+
+        # Forces the command to wait
+        self.get_command_result()
+
+    def set_property_values(self, prop_vals):
```

Comment:
I think we can just drop this method if we have the above and the individual `set_property`

Reply:
Done.